bundle/trampoline/python_wheel.go

```diff
@@ -61,22 +61,30 @@ s = f.getvalue()
 dbutils.notebook.exit(s)
 `
 
+type transformWheelTask struct{}
+
+func (transformWheelTask) Name() string {
+	return "TransformWheelTask"
+}
+
 // This mutator takes the wheel task and transforms it into notebook
 // which installs uploaded wheels using %pip and then calling corresponding
 // entry point.
-func TransformWheelTask() bundle.Mutator {
-	return bundle.If(
-		func(_ context.Context, b *bundle.Bundle) (bool, error) {
-			res := b.Config.Experimental != nil && b.Config.Experimental.PythonWheelWrapper
-			return res, nil
-		},
-		NewTrampoline(
-			"python_wheel",
-			&pythonTrampoline{},
-			NOTEBOOK_TEMPLATE,
-		),
-		mutator.NoOp(),
-	)
+func TransformWheelTask() transformWheelTask {
```

Comment:
should return `bundle.Mutator` instead

Reply:
I don't see why. Both would work though.

Reply:
Both would work, but it's to make the code consistent

Reply:
updated in f4f4988e642e02eab0d9c14f597a68fd7b0b68b2